unit/apply.sh

```diff
@@ -0,0 +1,64 @@
+#!/bin/sh
+
+if ! which jq >/dev/null; then
+	echo "jq is required. Aborting..."
+	exit 1
+fi
+
+. /etc/default/unit
+
+UNITPID=/var/run/unit.pid
+UNITSOCK=$UNITDIR/control.unit.sock
+
+if [ ! -f $UNITPID ]; then
+	echo "Unit is not running (File not found $UNITPID)"
+	exit 1
+fi
+
+if [ ! -S $UNITSOCK ]; then
+	echo "Socket file is not at $UNITSOCK"
+	exit 1
+fi
+
+CURL="curl --silent --unix-socket $UNITSOCK"
```

Comment:
![remember](https://i.imgflip.com/3dp53f.jpg)

Reply:
I dont get it haha

Reply:
It is just that to configure something you need curl/http/etc :P, was thinking on plan9 like stuff...where configuring things could be done with file operations etc =P (even dynamic ones). Sorry I will shut up =(